ecl/hql/hqlgram.y

```diff
@@ -111,6 +111,7 @@ static void eclsyntaxerror(HqlGram * parser, const char * s, short yystate, int
   ACOS
   AFTER
   AGGREGATE
+  ALGORITHM
```

Comment:
Needs to be added to ecl/eclcc/reservedwords.cpp
I'll add it to the simplify expected list.